RHDEVDOCS-4497:-  Release note for doc: Improve awareness of resource/project limits and quota issues for developers (#51338)
openshift/openshift-docs: release_notes/ocp-4-12-release-notes.adoc

* With this update, you can export your application in the ZIP file format to another project or cluster using the *Export application* option on the *+Add* page. 
* With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
* With this update, you can set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
* With this update, you can see the labels regarding quota status in the *Topology* view, which includes an alert to exceeding quotas.

> 

Reviewer: @smunje there are two parts to this epic:
Resource quota alert label on the Topology and Add page:
1. The above alert takes the user to the search page with all rqs and if there is just one rq then it takes the user to the details page for the same.

For deployments, if there are any errors associated with resource quotas, a warning alert is shown in the side panel.
Also a yellow border is shown around the deployment nodes when resource quota exceeds.

* With this update, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
* With this update, you can set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
* With this update, you can make the `status.HostIP` node IP address for pods visible in the *Details* tab of the *Pods* page.

> * With this update, you can see the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes the user to the *ResourceQuotas* search page having all resource quotas lists. If the alert label link is for a single resource quota, it takes the user to the *ResourceQuota details* page. 

Reviewer: ```suggestion
* With this update, you can see the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes the user to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes the user to the *ResourceQuota details* page. 
```

> ** For deployments, an alert is displayed in the topology node side panel if any errors are associated with resource quotas. Also, when the resource quota exceeds, a yellow border is displayed for the deployment nodes.

Reviewer: ```suggestion
** For deployments, an alert is displayed in the topology node side panel if any errors are associated with resource quotas. Also, when the resource quota exceeds, a yellow border is displayed around the deployment nodes.
```

* Create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
* Set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
* Make the `status.HostIP` node IP address for pods visible in the *Details* tab of the *Pods* page.

> * See the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes the user to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes the user to the *ResourceQuota details* page. 

Reviewer 2: ```suggestion
* See the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes you to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes you to the *ResourceQuota details* page. 
```

* Create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
* Set the default resource preference in the *User Preferences* -> *Applications* page. In addition, you can select another resource type to be the default.
* Make the `status.HostIP` node IP address for pods visible in the *Details* tab of the *Pods* page.
* See the resource quota alert label on the *Topology* and *Add* pages whenever any resource reaches the quota. The alert label link takes the user to the *ResourceQuotas* list page. If the alert label link is for a single resource quota, it takes the user to the *ResourceQuota details* page. 

> ** For deployments, an alert is displayed in the topology node side panel if any errors are associated with resource quotas. Also, when the resource quota exceeds, a yellow border is displayed for the deployment nodes.

Reviewer: ```suggestion
** For deployments, an alert is displayed in the topology node side panel if any errors are associated with resource quotas. Also, when the resource quota exceeds, a yellow border is displayed around the deployment nodes.
```
